### PR TITLE
Convert JS variables to JSON file

### PR DESCRIPTION
**Pull request by fResult**
Issue #4 krub

### CODE REVIEW

**Comment by narze** [comment]
That's neat!

By the way I think only 1 json file should be enough with nested format e.g.

```json
{
  "intro": { ... },
  "projects": [ ... ],
  ...
}
```

This way it'll be easier to load and export data in the future

**Comment by fResult** [comment]
> That's neat!
> 
> By the way I think only 1 json file should be enough with nested format e.g.
> 
> ```json
> {
>   "intro": { ... },
>   "projects": [ ... ],
>   ...
> }
> ```
> 
> This way it'll be easier to load and export data in the future

Fixed krub.\
I'm not sure this is correct yet.\
Please don't hesitate to suggest this to be better.

File: static/data/profile.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+	"interests": [
+		"Areas : Coding, Productivity & Life-hacks, Mechanical Keyboards",
+		"Typing : 120+ words per minute (Colemak Mod-DH layout)",
+		"Games : Music games, FPS, MOBA, Puzzle",
+		"Music : EDM, Rock, Japanese",
+		"Remote working is a OK. I extensively use self time tracking app for all my works.",
+		"Pets : I have 3 dogs"
+	],
+	"resumeUrl": {
+		"sourceLink": "https://resume.narze.live",
```
Comment (narze) [inline static/data/profile.json]: This line should link to github 
```suggestion
		"sourceLink": "https://github.com/narze/resume",
```

File: yarn.lock
Comment (narze) [inline yarn.lock:1]: Please remove yarn lockfile since this repo uses pnpm

File: static/data/profile.json (hunk 1 above)
Comment (narze) [reply static/data/profile.json]: image

the [Data] link should point to https://github.com/narze/resume/blob/main/static/data/profile.json (it'll work after merging to main)

**Comment by fResult** [comment]
Already fixed krub 😀

**Review by narze — APPROVED** [review]
💖

**Pull request merged**